service.py

```diff
@@ -144,7 +129,7 @@ def collect_initial_data():
         item_data['year'] = xbmc.getInfoLabel("ListItem.Year")
         item_data['season'] = xbmc.getInfoLabel("ListItem.Season")
         item_data['episode'] = xbmc.getInfoLabel("ListItem.Episode")
-        item_data['tvshow'] = xbmc.getInfoLabel("ListItem.TVShowTitle")
+        item_data['tvshow'] = getTVshowOriginalTitleByTMDBapi()
```

Comment:
It is better to pass params to this method instead of accessing global `xbmc` inside it?

Reply:
It this method works only for TVShows? what about movies?

Reply:
1. I don't think it makes any different. Keep it as it is.
If you think it's better calling them outside I will change that. This is your decision.

2. Regarding movies, the ListItem.TVShowTitle"" always returns the title in english, so there is no problem here as tv shows when "ListItem.TVShowTitle" return the title in Hebrew (for example) and "ListItem.TVShowTitle" returns empty string, so in order to make a string query the only option is to convert the Hebrew title to English (As implemented in this PR)
* Long time ago I could get the original title in English from kodi json api, but now it's not working anymore (probably changes in TVDB scraper or something), I reported on that.
* But I think it depends on the Movies scraper, mine is "Universal Movie Scraper".